### PR TITLE
US127858 [AQ] New Temporary Assessment Quality Dashboard Page. Initial page implementation

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
[US127858](https://rally1.rallydev.com/#/23525809118d/portfolioitemstreegrid?detail=%2Fuserstory%2F603047932448&fdp=true?fdp=true): [AQ] New Temporary Assessment Quality Dashboard Page

FYI @anhill-D2L @hyehayes @NicholasHallman

This PR contains:
* basic page implementation 
* initial version of package.json with copied dependencies from https://github.com/Brightspace/insights-engagement-dashboard
* simple test for the page
* ci.yml
* make notice that some files like README.md, LICENSE, .gitignore, .eslintrc, .editorconfig, CODEOWNERS, basic package.json have been added directly to the master. Sorry for that.

To be done in a separate PR:

- [ ] immersive nav
- [ ] visual-diff

### Page screenshot

image

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
+      - name: Linting
+        run: npm run lint
+  sauce:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Brightspace/third-party-actions@actions/checkout
+      - uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+          node-version: 12.x
```
Comment (bemailloux) [inline .github/workflows/ci.yml:22]: Does it work on node 14? Might as well update if we can

File: package.json [hunk 2]
```diff
@@ -5,12 +5,65 @@
...
+    "d2l-fetch-auth": "^1",
+    "d2l-navigation": "BrightspaceUI/navigation#semver:^4",
+    "d2l-table": "BrightspaceUI/table#semver:^2",
+    "d2l-telemetry-browser-client": "Brightspace/d2l-telemetry-browser-client#semver:^1",
+    "d2l-users": "BrightspaceHypermediaComponents/users#semver:^2",
+    "export-from-json": "^1.3.4",
+    "highcharts": "^9.1.0",
+    "lit-element": "^2",
+    "lit-html": "^1.3.0",
+    "mobx": "^5.15.4"
```
Comment (bemailloux) [inline package.json:67]: Might be a good idea to only start with the dependencies we know we'll need

**Review by bemailloux — APPROVED** [review]

File: locales/localizer.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+						break;
+					case 'es':
+						translations = await import('./es.js');
+						break;
+					case 'es-es':
+						translations = await import('./es-es.js');
+						break;
+					case 'fr':
+						translations = await import('./fr.js');
+						break;
```
Comment (johngwilkinson) [inline locales/localizer.js:50]: I imagine we'll need fr-ON too

**Review by johngwilkinson — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Vitalii-Misechko) [reply .github/workflows/ci.yml]: > Does it work on node 14? Might as well update if we can

It does. Update ci.yml

File: package.json (hunk 2 above)
Comment (Vitalii-Misechko) [reply package.json]: > Might be a good idea to only start with the dependencies we know we'll need

Thanks. I removed unused dependencies

File: locales/localizer.js (hunk 3 above)
Comment (Vitalii-Misechko) [reply locales/localizer.js:50]: > I imagine we'll need fr-ON too

Good point. Added fr-ON into `localizer.js`

**Pull request merged**